### PR TITLE
feat: merge login and intro page

### PR DESCRIPTION
**Pull request by mrdrogdrog**
### Component/Part
frontend

### Description
This PR merges the login and intro page

### Steps

- [x] Added implementation
- [x] Added / updated tests
- [x] I read the [contribution documentation](https://github.com/hedgedoc/hedgedoc/blob/develop/CONTRIBUTING.md) and
  made sure that:
  - My commits are signed-off to accept the DCO.
  - This PR targets the correct branch: `master` for 1.x & docs, `develop` for 2.x

### CODE REVIEW

File: frontend/src/app/(editor)/@appBar/login/page.tsx [hunk 1]
```diff
@@ -1,11 +1,29 @@
...
+      <Nav className={`align-items-stretch justify-content-end flex-grow-1 ${styles.side} h-100 py-1`}>
+        <div className={'d-flex gap-2'}>
+          <HistoryButton />
+          <HelpDropdown />
+          <SettingsButton />
+          <NewNoteButton />
+        </div>
+      </Nav>
+    </Navbar>
+  )
```
Comment (DerMolly) [inline frontend/src/app/(editor)/@appBar/login/page.tsx:28]: This is a duplicate of the code in `base-app-bar.tsx`, isn't it? Is this intended? If yes why is this necessary?

Comment (mrdrogdrog) [reply frontend/src/app/(editor)/@appBar/login/page.tsx:28]: No. This one has no branding, no hedgedoc icon and no user menu.

**Review by DerMolly — APPROVED** [review]

**Review by ErikMichelson — APPROVED** [review]
